kie-server-parent/kie-server-api/src/test/java/org/kie/server/api/marshalling/objects/DateObject.java

```diff
@@ -0,0 +1,77 @@
+package org.kie.server.api.marshalling.objects;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.OffsetDateTime;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.adapters.XmlJavaTypeAdapter;
+
+import com.thoughtworks.xstream.annotations.XStreamAlias;
+import com.thoughtworks.xstream.annotations.XStreamConverter;
+import org.kie.internal.jaxb.LocalDateTimeXmlAdapter;
+import org.kie.internal.jaxb.LocalDateXmlAdapter;
+import org.kie.internal.jaxb.LocalTimeXmlAdapter;
+import org.kie.internal.jaxb.OffsetDateTimeXmlAdapter;
+import org.kie.internal.xstream.LocalDateTimeXStreamConverter;
+import org.kie.internal.xstream.LocalDateXStreamConverter;
+import org.kie.internal.xstream.LocalTimeXStreamConverter;
+import org.kie.internal.xstream.OffsetDateTimeXStreamConverter;
+
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlRootElement(name = "date-object")
+@XStreamAlias( "date-object" )
+public class DateObject {
+
+    @XmlJavaTypeAdapter(LocalDateXmlAdapter.class)
+    @XStreamConverter(LocalDateXStreamConverter.class)
```

Comment:
Please add a TODO comment with the url to the xstream PR or issue that will replace our version of the LocalDateXStreamConverter